### PR TITLE
Lms 25.connect.some.library.to.log.errors

### PR DESCRIPTION
**Pull request by MikolaLvivUA**

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -85,6 +93,7 @@ class App {
             next(err);
         }
     }
+
```
Comment (VictorKmin) [inline src/app.ts]: rm

File: src/configs/winston.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import * as winston from 'winston';
+
+const options = {
+    errorFile: {
+        level: 'error',
+        name: 'file.error',
+        filename: `${__dirname}/../logs/error.log`,
```
Comment (VictorKmin) [inline src/configs/winston.ts]: maybe will be better to use appRoot from app.ts?

File: src/configs/winston.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        handleExceptions: true,
+        json: true,
+        maxsize: 5242880, // 5MB
```
Comment (VictorKmin) [inline src/configs/winston.ts]: to constant
5*1024*1024

File: src/controllers/download/download.controller.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { NextFunction, Request, Response } from 'express';
+
+class DownloadController {
+    async ErrorLogFileDownload(req: Request, res: Response, next: NextFunction) {
+        try {
+            const file = `${__dirname}../../../logs/error.log`;
```
Comment (VictorKmin) [inline src/controllers/download/download.controller.ts]: The same

File: src/routes/download/download.router.ts [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
+import { downloadController } from '../../controllers/download';
```
Comment (VictorKmin) [inline src/routes/download/download.router.ts]: ../../constrollers
Empty line

File: src/validators/insertedQuestionValidator.ts [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import * as Joi from 'joi';
 
-import {QuestionLevelEnum} from "../constants/enums";
+import { QuestionLevelEnum } from '../constants/enums';
```
Comment (VictorKmin) [inline src/validators/insertedQuestionValidator.ts]: ../constants

File: src/app.ts (hunk 1 above)
Comment (MikolaLvivUA) [reply src/app.ts]: fixed

File: src/configs/winston.ts (hunk 3 above)
Comment (MikolaLvivUA) [reply src/configs/winston.ts]: fixed

File: src/routes/download/download.router.ts (hunk 5 above)
Comment (MikolaLvivUA) [reply src/routes/download/download.router.ts]: fixed

File: src/validators/insertedQuestionValidator.ts (hunk 6 above)
Comment (MikolaLvivUA) [reply src/validators/insertedQuestionValidator.ts]: fixed

**Pull request merged**